install/kubernetes/helm/istio/charts/security/templates/tests/test-citadel-connection.yaml

```diff
@@ -20,7 +20,7 @@ spec:
 {{- end }}
   containers:
     - name: "{{ template "security.fullname" . }}-test"
-      image: {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
+      image: istionightly/base_debug
```

Comment:
This doesn't look right? Is this supposed to be changed?

Reply:
> This doesn't look right? Is this supposed to be changed?

What exactly do you mean? This is only used for a curl for testing purposes. I don't see why this should use TAG/HUB.

Reply:
@c0d1ngm0nk3y its a bit surprising to see.  A comment would help or perhaps what would make more sense is to break out a separate PR Rfor all of the testing charts and replace the images with the curl pod.

Cheers
-steve